Sources/Paywalls/PaywallData+Localization.swift

```diff
@@ -50,13 +53,7 @@ public extension PaywallData {
     /// - Returns: The list of locales that paywalls should try to search for.
     /// Includes `Locale.current` and `Locale.preferredLanguages`.
     internal static var localesOrderedByPriority: [Locale] {
-        var result = [.current] + Locale.preferredLocales
-
-        if let withoutRegion = Locale.current.removingRegion {
-            result.append(withoutRegion)
-        }
-
-        return result
+        return [.current] + Locale.preferredLocales
```

Comment:
This goes back to the original implementation. It allows us to test the actual behavior in `localizedConfiguration` since it no longer relies on this returning the `.removingRegion` addition.